00_Base/src/services/TokensAdminService.ts

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { Service } from 'typedi';
+import { OcpiLogger } from '../util/logger';
+import { TokensRepository } from '../repository/TokensRepository';
+import { AsyncJobName, AsyncJobStatus } from '../model/AsyncJobStatus';
+import { TokensClientApi } from '../trigger/TokensClientApi';
+import { buildPaginatedOcpiParams } from '../trigger/param/paginated.ocpi.params';
+import { Role } from '../model/Role';
```

Comment:
github linter warning :eyes: -- 'Role' is defined but never used. Allowed unused vars must match /^_/u